### PR TITLE
Upgrade ansible to 2.0.0.1

### PR DESCRIPTION
**Pull request by acaire**
Upgrade Paramiko, missed from https://github.com/Homebrew/homebrew/pull/48003

Upgrade pywinrm dependencies as per: http://docs.ansible.com/ansible/intro_windows.html#installing-on-the-control-machine

Add boto3 support for new extras modules

### CODE REVIEW

File: Library/Formula/ansible.rb [hunk 1]
```diff
@@ -156,13 +156,42 @@ class Ansible < Formula
...
   #
-  # boto (for AWS support)
+  # boto/boto3 (for AWS support)
   #
   resource "boto" do
     url "https://pypi.python.org/packages/source/b/boto/boto-2.38.0.tar.gz"
     sha256 "d9083f91e21df850c813b38358dc83df16d7f253180a1344ecfedce24213ecf2"
   end
 
+  resource "boto3" do
```
Comment (dunn) [inline Library/Formula/ansible.rb]: What modules are these new dependencies required for?  If they're ones that should be enabled by default, add `revision 1` underneath line 5 so everyone updates.

Comment (acaire) [inline Library/Formula/ansible.rb]: [ecs_cluster](http://docs.ansible.com/ansible/ecs_cluster_module.html)
[ecs_task](http://docs.ansible.com/ansible/ecs_task_module.html)
[ec2_taskdefinition](http://docs.ansible.com/ansible/ecs_taskdefinition_module.html)
[route53_facts](http://docs.ansible.com/ansible/route53_facts_module.html)

Not sure how to interpret 'should be enabled by default' - The module is available to use. if the user chooses to use it.

Comment (dunn) [inline Library/Formula/ansible.rb]: > Although Ansible have since released 2.0.0.1 - I could just upgrade to that?

Yes, please!

**Comment by acaire** [comment]
Bumped to 2.0.0.1, thanks.

**Comment by dunn** [comment]
Thanks so much! Merged in 45955bf6dea94339ea9020899b6e22d9154a79df.

**Pull request closed without merging**